crates/egui/src/util/cache.rs

```diff
@@ -39,7 +39,7 @@ impl<Value, Computer> FrameCache<Value, Computer> {
     }
 
     /// Must be called once per frame to clear the cache.
-    pub fn evice_cache(&mut self) {
+    pub fn evict_cache(&mut self) {
         let current_generation = self.generation;
         self.cache.retain(|_key, cached| {
             cached.0 == current_generation // only keep those that were used this frame
```

Comment:
This is a breaking change but as far as I can tell this is only called in CacheTrait::update and no external code calls this ([confirmed via github code search](https://github.com/search?q=%2F%5CWevice_cache%2F&type=code))